### ISSUE TITLE
All Apps suddenly disappeared without data loss

### ISSUE THREAD

**Issue by Smoophyy** [GrapheneOS/os-issue-tracker#2071]
Hi Everyone,

First things first. I have a Pixel 7 Pro which I have been using for about 2 Months.
Build Number is TQ2A.230305.008.C1.2023032600
Using just one profile with Shelter to seperate things like WhatsApp.

Didn't have any real problems before. I rebooted my device today and it took 15+ Minutes.
I decided to force a shutdown (I think it's volume up + Power).
Started it and it did just fine. So I unlocked, had Internet and then put the phone in my car and drove off.
5 Minutes later I saw the bootscreen on the phon even tho I have not touched it.
It started and I unlocked it, typed in the simcode and was greeted by my usual start screen minus ALL non stock apps.
In both private and work profile all apps are gone. But the files are still there when I use the file explorer.
My contacts are also wiped clean. nothing on there.
I can't connect to any service with both physical and e-sim.

Is this issue known? Is there anything I can do? I will not use the phone just to make sure I don't change anything.

Thanks!

PS: Also posted this on here https://discuss.grapheneos.org/d/4232-all-apps-suddenly-gone-but-data-still-there-pls-help/2

**Comment by thestinger** [comment]
This is the first report of anything like this. You're using a device manager (Shelter) which could be the cause. The issue could also be that you interrupted the boot process during an important phase in a way that Android doesn't handle properly. Not really anything we can do without further information to determine what could have been the issue.

**Comment by thestinger** [comment]
Would help if you clarified what you mean by no data loss. Apps have their own data directories.

https://grapheneos.org/usage#storage-access

If you reinstall the apps, do you get back their app data, or is it gone?

**Comment by Smoophyy** [comment]
Okay so I reinstalled Audbile and Whastapp Business.

Audible shows no installed audiobooks and if I try to install them I am getting the an error "download failed"
If I do go to Android -> Data I see the folders of the apps I had installed. Some of them are empty (for example discord only has a folder called "files" which is empty).
Some fodlers, like Telegramm are still full of data. But just like Whatsapp it wants me to log in with my number but since I do not have any service I won't be able to receive the SMS-Code.

Android -> Media only contains WhatsApp and Facebook. Whatsapp is still full with data, facebook is not (not using it except for business things so not sure if there should even be something in there)

OBB contains the Hearthstone folder but is also empty as expected
